src/openrct2/Context.cpp

```diff
@@ -256,6 +257,10 @@ namespace OpenRCT2
             {
                 auto dataPath = _env->GetDirectoryPath(DIRBASE::RCT1, DIRID::DATA);
                 result = Path::Combine(dataPath, "css17.dat");
+                if (!(std::ifstream(result.c_str()).good())){
```

Comment:
You should not open a stream on a file just to test its existence.

Reply:
Then how would you do it?

Reply:
We have a `File::Exists` helper, use that.

Reply:
Ok. Thx @IntelOrca.